packages/harpguru-core/src/components/harp-faces/harp-faces.tsx

```diff
@@ -0,0 +1,44 @@
+import { useGlobal } from 'reactn'
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+import { isChromaticHarpFace } from 'harpparts'
+
+import { HarpFace } from '../harp-face'
+import { useSizes } from '../../hooks'
+
+export const HarpFaces = (): React.ReactElement => {
+  const [fullInteractionMatrix] = useGlobal('activeInteractionMatrix')
+  const {
+    dynamicSizes: { fragmentGutter },
+  } = useSizes()
+  const { stack, gutter } = StyleSheet.create({
+    stack: {
+      ...StyleSheet.absoluteFillObject,
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    gutter: {
+      height: fragmentGutter,
+    },
+  })
+  // TODO: Have this as part of the layout facts
+  // hook and use both here and in the use sizes
+  // hook.
+  if (isChromaticHarpFace(fullInteractionMatrix)) {
+    return (
+      <View style={stack}>
+        <HarpFace harpfaceIndex={'harpface1'} />
+        <View style={gutter} />
+        <HarpFace harpfaceIndex={'harpface2'} />
+      </View>
+    )
+  }
+  return (
+    <View style={stack}>
+      <HarpFace harpfaceIndex={'harpface1'} />
+    </View>
+  )
+}
+
+// TODO: memoise this component to only return a new one if the number of harpfaces changes
```

Comment:
After some experimentation it doesn't look like there is much to be gained here.

Reply:
That's not true, but issues have now been raised - see https://github.com/js-jslog/harpguru/pull/129#discussion_r761850991